.github/workflows/on_target.yml

```diff
@@ -202,6 +230,76 @@ jobs:
         env:
           SEGGER: ${{ secrets.SEGGER_DUT_1 }}
 
+      - name: Run PPK tests
+        if: ${{ inputs.run_ppk_tests }}
+        id: ppk_test
+        working-directory: thingy91x-oob/tests/on_target
+        run: |
+          mkdir -p results
+          pytest -s -v -m dut_ppk \
+            --junit-xml=results/test-results-ppk.xml \
+            tests
+        env:
+          SEGGER: ${{ secrets.SEGGER_DUT_PPK }}
+
+      - name: Commit and Push Badge File to gh-pages Branch
+        if: always()
+        working-directory: thingy91x-oob
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
```

Comment:
Consider moving this to a seperate bash script since it's quite much commands.

Reply:
will do